uber/config.py

```diff
@@ -363,6 +363,8 @@ def _is_intstr(s):
 
 c.TABLE_PRICES = defaultdict(lambda: _config['table_prices']['default_price'],
                              {int(k): v for k, v in _config['table_prices'].items() if k != 'default_price'})
```

Comment:
Out of curiosity, why this change?  A plugin can say ``c.MAX_TABLES = 123`` to modify this config just as easily as they can modify ``c.PREREG_TABLE_OPTS``.

Relatedly, if you're already fixing the admin table value, maybe make it configurable while you're at it?  Not a huge deal if you're too swamped and don't have time, but if you're making changes anyway...